.drone.jsonnet

```diff
@@ -420,10 +420,17 @@ local integration_extensions = Step('e2e-extensions', target='e2e-qemu', privile
   WITH_TEST: 'run_extensions_test',
   IMAGE_REGISTRY: local_registry,
 });
-local integration_cilium = Step('e2e-cilium-1.9.10', target='e2e-qemu', privileged=true, depends_on=[integration_extensions], environment={
+local integration_cilium = Step('e2e-cilium', target='e2e-qemu', privileged=true, depends_on=[integration_extensions], environment={
```

Comment:
I feel like we should probably split `integration-misc` a bit, as it now might run for 1.5 hours with the `extensions` build... nothing pressing, probably `e2e-extensions` should be its own pipeline, as it got significantly more complex?

Reply:
yes, makes sense, should cilium also be it's own pipeline, also would make my testing easier :sweat_smile:

Reply:
as I was testing this, I commented out everything but Cilium from the `integration-misc`, or make it the first one - the order doesn't matter

Reply:
> as it now might run for 1.5 hours with the extensions build

extensions e2e is a different pipeline, it's only part of intergration-misc as part of cron, should we change it to a separate cron?

Reply:
yes, probably